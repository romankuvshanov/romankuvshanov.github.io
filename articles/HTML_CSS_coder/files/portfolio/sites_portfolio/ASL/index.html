<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>ASL</title>
    <link rel="stylesheet" href="css/main.css">
</head>
<body>
    <section class="main">
        <div class="container">
            <header class="flex">
                <img src="img/logo.png" alt="Logo">
                <nav>
                    <ul>
                        <li><a href="#">ТАРИФЫ</a></li>
                        <li><a href="#">О НАС</a></li>
                        <li><a href="#">НАШИ УСЛУГИ</a></li>
                        <li><a href="#">Почему мы?</a></li>
                        <li><a href="#">Контакты</a></li>
                    </ul>
                </nav>
                <p class="contacts">asldir@mail.ru<span>+7 700 700 59 59</span></p>
            </header>
            <h1>Пассажирские перевозки<br>от 4000 тенге/час<br><span>на абсолютно новых Hyundai H1</span></h1>
            <p>Оставьте заявку,<br>чтобы получить консультацию</p>
            <a href="#" class="outline-button">ЗАКАЗАТЬ ЗВОНОК</a>
            <p>и мы свяжемся с вами в ближайшее время</p>
        </div>
    </section>
    <section class="tariffs">
        <div class="container">
            <h1 class="title-with-back">Тарифы</h1>
            <div class="grid">
                <div class="events">
                    <h2>ОБСЛУЖИВАНИЕ МЕРОПРИЯТИЙ,<br>
                    КОНФЕРЕНЦИЙ ПО ГОРОДУ,<br>
                    ВСТРЕЧА И ПРОВОДЫ В АЭРОПОРТУ</h2>
                    <img src="img/events.png" alt="">
                    <p><span class="price">4000 тг/час</span><br>Минимальный заказ не менее 2-часов</p>
                </div>
                <div class="rides">
                    <h2>МЕЖДУГОРОДНИЕ<br>
                    ПАССАЖИРСКИЕ ПЕРЕВОЗКИ</h2>
                    <img src="img/rides.png" alt="">
                    <p>
                        <span class="price">100 тг/км</span> - при заказе с обратным трансфером<br>
                        <span class="price">130 тг/км</span> - при заказе трансфера в одну сторону
                    </p>
                    <p class="about-ride">
                        ВСЕ ПЕРЕВОЗКИ ОСУЩЕСТВЛЯЮТСЯ НА НОВЫХ И
                        КОМФОРТАБЕЛЬНЫХ МИКРОАВТОБУСАХ HUNDAI H1
                        (8 пассажирских мест)
                    </p>
                </div>
            </div>
        </div>
    </section>
    <section class="video">
        <div class="container">
            <div class="parallelepipeds">
                <div class="medium"></div>
                <div class="large"></div>
                <div class="small"></div>
            </div>
            <div class="content">
                <h1>Посмотрите видео, чтобы подробнее ознакомиться с<br>
                <span>Astana Service Logistic</span></h1>
                <div class="grid">
                    <div class="presentation">
                        <h2>ПРЕЗЕНТАЦИЯ КОМПАНИИ</h2>
                        <iframe width="560" height="315" src="https://www.youtube.com/embed/RuH1EktkAlQ" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
                    </div>
                    <div class="tv">
                        <iframe width="560" height="315" src="https://www.youtube.com/embed/RuH1EktkAlQ?start=50" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
                        <h2>ВИДЕОСЮЖЕТ НА ТЕЛЕКАНАЛЕ «ASTANA»</h2>
                    </div>
                </div>
            </div>
        </div>
    </section>
    <section class="know-more">
        <div class="container">
            <h1>Хотите узнать больше?</h1>
            <h2>Наши сотрудники с удовольствием<br>ответят на любой Ваш вопрос!</h2>
            <form class="flex">
                <input type="text" name="name" placeholder="Введите Ваше Имя">
                <input type="tel" name="phone" placeholder="Введите Ваш телефон*">
                <button type="submit" class="outline-button">ЗАКАЗАТЬ ЗВОНОК</button>
            </form>
            <p>*обязательное поле</p>
        </div>
    </section>
    <section class="our-services">
        <div class="container">
            <h1 class="title-with-back">Тарифы</h1>
            <img src="img/services.png" alt="">
        </div>
    </section>
    <section class="compare">
        <div class="container">
            <h1>Нелегальные перевозчики Vs <span>Astana Service Logistic</span></h1>

           <div class="grid titles">
               <h2>Нелегальные перевозчики:</h2>
               <h2>Astana Service Logistic:</h2>
           </div>
            <div class="arguments grid">
                <div class="ilegal">
                    
                    <ul>
                        <li>- Низкий профессионализм водителей</li>
                        <li>- Безконтрольность прохождения ТО автомашин</li>
                        <li>- Полное отсутствие медицинского допуска к рейсу</li>
                        <li>- Не существует регламента посадки и высадки пассажиров</li>
                        <li>- Увеличение уровня потенциальной опасности на дорогах<br>и уровня теневой экономики</li>
                    </ul>
                </div>
                <div class="asl">
                    <ul>
                        <li>- Обучение и переподготовка водителей</li>
                        <li>- Ежедневный технический осмотр автомобилей</li>
                        <li>- Регулярный медосмотр водителей перед каждой поездкой</li>
                        <li>- Регламентированная безопасная посадка и высадка пассажиров</li>
                        <li>- Формирование легального рынка в сфере экскурсионных,<br>туристических и заказных перевозок</li>
                    </ul>
                </div>
            </div>
        </div>
    </section>
    <section class="our-goal">
        <div class="container">
            <h1><span>Наша цель</span> - формирование легального<br>
            и упорядоченного рынка пассажирских перевозок</h1>
            <div class="benefits flex">
                <div class="benefit">
                    <img src="img/take_off.png" alt="Sheet icon">
                    <p>Регламентированная<br>посадка и высадка пассажиров</p>
                </div>
                <div class="benefit">
                    <img src="img/bus.png" alt="Bus icon">
                    <p>Ежедневный<br>технический осмотр машин</p>
                </div>
                <div class="benefit">
                    <img src="img/stethoscope.png" alt="Stethoscope icon">
                    <p>Регулярный медосмотр<br>водителей перед каждой поездкой</p>
                </div>
            </div>
        </div>
    </section>
    <footer>
        <div class="container flex">
            <p class="text-left">
                asldir@mail.ru<br>
                Астана, ул. Елубая Тайбекова, д. 63
            </p>
            <div class="text-right">
                <p>ОСТАЛИСЬ ВОПРОСЫ? ЗВОНИТЕ!<br>
                <span>+7 700 700 59 59</span></p>
                <a href="#" class="outline-button">ЗАКАЗАТЬ ЗВОНОК</a>
            </div>
        </div>
    </footer>
</body>
</html>