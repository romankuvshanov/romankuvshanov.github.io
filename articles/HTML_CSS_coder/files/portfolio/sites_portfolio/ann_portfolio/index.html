<!DOCTYPE html>
<html lang="ru">
    <head>
        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link href="https://fonts.googleapis.com/css?family=Exo+2:400,700" rel="stylesheet">
        <link rel="stylesheet" href="css/style.css">
        <link rel="icon" type="image/png" href="img/favicons/favicon-16x16.png" sizes="16x16" data-mce-href="/favicon-16x16.png">
        <link rel="icon" type="image/png" href="img/favicons/favicon-32x32.png" sizes="32x32" data-mce-href="/favicon-32x32.png">
        <link rel="icon" type="image/png" href="img/favicons/favicon-96x96.png" sizes="96x96" data-mce-href="/favicon-96x96.png">
        <link rel="icon" type="image/png" href="img/favicons/android-chrome-192x192.png" sizes="192x192" data-mce-href="/android-chrome-192x192.png">
        <title>Anna Blok - portfolio</title>
    </head>
    <body>
        <header>
            <div class="container">
                <img src="img/portrait_round.png" alt="My portrait in round frame">
                <h1>Hello, my name is Anna and I create cool websites</h1>
                <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud</p>
                <a href="#" class="hire-me-button">Hire me</a>
                <a href="#scroll" id="arrow-down"></a>
            </div>
        </header>
        
        <section class="block-scroll">
            <div class="container">
                <h2>About Me</h2>
                <p>
                Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia 
                </p>
            </div>
        </section>
        
        <section class="blue">
            <div class="container">
                <h2>Expert In</h2>
                
                <div class="flexbox">
                    <div class="block">
                        <div class="icon" id="mobile-app-icon">
                        
                        </div>
                        <h3>MOBILE APP DESIGN</h3>
                        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.  Ut enim ad</p>
                    </div>
                
                
                
                    <div class="block">
                        <div class="icon" id="web-design-icon">
                        
                        </div>
                        <h3>WEB DESIGN</h3>
                        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
                    </div>
                
                
                
                    <div class="block">
                        <div class="icon" id="web-development-icon">
                        
                        </div>
                        <h3>WEB DEVELOPMENT</h3>
                        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.  Ut enim ad</p>
                    </div>
                </div>
            </div>
        </section>
        
        <section>
            <div class="container">
                <h2>Latest Works</h2>
                
                <div class="flexbox">
                    <div class="block">
                        <div class="img-wrapper">
                            <a href="#"><img src="img/work_1.jpg" alt="Man with sunflower head"></a>
                        </div>
                    </div>
                
                
                
                    <div class="block">
                        <div class="img-wrapper">
                            <a href="#"><img src="img/work_2.jpg" alt="American roadway with palms on the sidewalks"></a>
                        </div>
                    </div>
                
                
                
                    <div class="block">
                        <div class="img-wrapper">
                            <a href="#"><img src="img/work_3.jpeg" alt="Sky art"></a>
                        </div>
                    </div>
                </div>
                
            </div>
        </section>
        
        <footer>
            <div class="container flexbox">
                <p>Need a web developer? <span>Ready to work!</span></p>
                <a href="#" class="hire-me-button">Hire me</a>
            </div>
            <a class="btn-up" href="#"></a>
        </footer> 
        
        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
        <script type="text/javascript">
            $(function(){ 
                $(window).scroll(function(){ 
                    if($(this).scrollTop()!=0){ 
                        $('.btn-up').fadeIn(); 
                    } 
                    else{ 
                        $('.btn-up').fadeOut(); 
                    } 
                }); 
                $('.btn-up').click(function(){ 
                    $('body,html').animate({scrollTop:0},500); 
                }); 
            });
            
            $(function() { 
                $('a[href="#scroll"]').click(function(event) { 
                    event.preventDefault() 
                    var el = $('.block-scroll'); 
                    $('body,html').animate({ 
                        scrollTop: $(el).offset().top 
                    }, 1000); 
                }); 
            }); 
</script>
    </body>
</html>