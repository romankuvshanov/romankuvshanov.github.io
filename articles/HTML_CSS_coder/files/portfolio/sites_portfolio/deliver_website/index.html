<!DOCTYPE html>
<html lang="ru">
    <head>
        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.0.13/css/all.css" integrity="sha384-DNOHZ68U8hZfKXOrtjWvjxusGo9WQnrNx2sqG0tfsghAvtVlRW3tvkXWZh58N9jp" crossorigin="anonymous">
        <link href="https://fonts.googleapis.com/css?family=Poppins:300,400,500,600,700" rel="stylesheet">
        <link href="css/main.css" rel="stylesheet">
        <link rel="icon" type="image/png" href="img/favicons/favicon-16x16.png" sizes="16x16" data-mce-href="/favicon-16x16.png">
        <link rel="icon" type="image/png" href="img/favicons/favicon-32x32.png" sizes="32x32" data-mce-href="/favicon-32x32.png">
        <link rel="icon" type="image/png" href="img/favicons/favicon-96x96.png" sizes="96x96" data-mce-href="/favicon-96x96.png">
        <link rel="icon" type="image/png" href="img/favicons/android-chrome-192x192.png" sizes="192x192" data-mce-href="/android-chrome-192x192.png">
        <title>Deliver Website</title>
    </head>
    <body>
        <header>
                <nav id="language-and-country">
                    <div class="container">
                        <select>
                            <option>English</option>
                            <option>Русский</option>
                            <option>Español</option>
                        </select>
                        <div id="contact-center-and-contry">
                            <a href="#"><i class="fas fa-users"></i> Contact Center</a>
                            <select>
                                <option>Country Profile</option>
                                <option>English</option>
                                <option>Русский</option>
                                <option>Español</option>
                            </select>
                        </div>
                    </div>
                </nav>
                <nav id="express-parcel">
                    <div class="container">
                        <img src="img/dhl_logo.png" alt="DHL logo">
                        <ul>
                            <li><a href="#">Express</a></li>
                            <li><a href="#">Parcel & Ecommerce</a></li>
                            <li class="current"><a href="#">Logistics</a></li>
                            <li><a href="#">Mail</a></li>
                            <li><a href="#">Press</a></li>
                            <li><a href="#">About</a></li>
                        </ul>
                        <a href="#" class="search"><i class="fas fa-search"></i></a>
                    </div>
                </nav>
        </header>
        <section id="we-deliver">
            <div class="container">
                <h1>We Deliver<br>With Speed</h1>
                <div id="plate"></div>
                <p>When an unknown printer took a galley of type and scrambled
                   it to make a type specimen book. It has survived not only five centuries</p>
                <p id="qualities">Amazing<span>·</span>Quick<span>·</span>24/7 Supports</p>
                <a href="#" class="outline-btn">Learn More</a>
                <a href="#" class="outline-btn">Dhl worldwide</a>
            </div>
        </section>
        <section id="services">
            <div class="container">
                <h3>Services</h3>
                <h1>Awesome Services</h1>
                <div class="flex-container">
                    <div class="flex-item">
                        <img src="img/service_1.jpg" alt="Мужчина выгружает коробки с надписями DHL">
                        <h2>Express Services</h2>
                        <p>When an unknown printer took a galley of type and scrambled it to make a type specimen.</p>
                        <a href="#">Read More <i class="fas fa-caret-right"></i></a>
                    </div>
                    <div class="flex-item">
                        <img src="img/service_2.jpg" alt="Курьер DHL с банковским терминалом на улице">
                        <h2>Parcel & Ecommerce</h2>
                        <p>When an unknown printer took a galley of type and scrambled it to make a type specimen.</p>
                        <a href="#">Read More <i class="fas fa-caret-right"></i></a>
                    </div>
                    <div class="flex-item">
                        <img src="img/service_3.jpg" alt="Курьер DHL едет на мотороллере">
                        <h2>Frieght transport</h2>
                        <p>When an unknown printer took a galley of type and scrambled it to make a type specimen.</p>
                        <a href="#">Read More <i class="fas fa-caret-right"></i></a>
                    </div>
                    <div class="flex-item" id="location-search">
                        <h2>DHL Worldwide</h2>
                        <div class="plate"></div>
                        <p>When an unknown prot printer type and scra</p>
                        <div id="select-and-chevron">
                            <select>
                                    <option>Location</option>
                                    <option>English</option>
                                    <option>Русский</option>
                                    <option>Español</option>
                            </select>
                            <i class="fas fa-chevron-down"></i>
                        </div>
                        <a href="#" class="outline-btn">Explore</a>
                    </div>
                </div>
            </div>
        </section>
        <section id="video">
            <video src="img/This%20is%20DHL%20Express.mp4"></video>
            <h1>One of the best international service provider</h1>
        </section>
        <section id="delivery-types">
            <div class="container">
                <div class="flex-container">
                    <div class="flex-item">
                        <img src="img/delivery_type_1.jpg" alt="Двое мужчин что-то обсуждают на фоне грузовика DHL">
                        <h1>Aerospace Provider</h1>
                        <p><i class="fas fa-caret-right"></i> When an unknown printer took.</p>
                        <p><i class="fas fa-caret-right"></i> Make a type specimen book.</p>
                        <p><i class="fas fa-caret-right"></i> It has survived.</p>
                        <p><i class="fas fa-caret-right"></i> Airlines unknown specimen.</p>
                    </div>
                    <div class="flex-item">
                        <img src="img/delivery_type_2.jpg" alt="Много грузовиков DHL">
                        <h1>Poppins Proved</h1>
                        <p><i class="fas fa-caret-right"></i> When an unknown printer took.</p>
                        <p><i class="fas fa-caret-right"></i> Make a type specimen book.</p>
                        <p><i class="fas fa-caret-right"></i> It has survived.</p>
                        <p><i class="fas fa-caret-right"></i> Airlines unknown specimen.</p>
                    </div>
                    <div class="flex-item">
                        <img src="img/delivery_type_3.jpg" alt="Самолёт DHL">
                        <h1>Airline Specimen</h1>
                        <p><i class="fas fa-caret-right"></i> When an unknown printer took.</p>
                        <p><i class="fas fa-caret-right"></i> Make a type specimen book.</p>
                        <p><i class="fas fa-caret-right"></i> It has survived.</p>
                        <p><i class="fas fa-caret-right"></i> Airlines unknown specimen.</p>
                    </div>
                </div>
            </div>
        </section>
        <section id="img-and-menu">
            <div class="container">
                <div id="menu">
                    <div class="tab">
                        <input type="radio" id="tab1" name="tab-group" checked>
                        <label for="tab1" class="tab-title">Express</label>
                        <section class="tab-content">
                            <div class="item">
                                <input type="checkbox" id="item1" name="item-group" checked>
                                <label for="item1" class="item-title">Track your shipment <i class="fas fa-chevron-down"></i></label>
                                <section class="item-content"> 
                                    <h1>International Services to Another Country</h1>
                                    <p><i class="fas fa-caret-right"></i>Emergency</p>
                                    <p><i class="fas fa-caret-right"></i>Time Critical</p>
                                    <p><i class="fas fa-caret-right"></i>Less Urgent</p>
                                </section>
                            </div>
                            <div class="item">
                                <input type="checkbox" id="item2" name="item-group">
                                <label for="item2" class="item-title">New login <i class="fas fa-chevron-down"></i></label>
                                <section class="item-content"> 
                                    <h1>International Services to Another Country</h1>
                                    <p><i class="fas fa-caret-right"></i>Emergency</p>
                                    <p><i class="fas fa-caret-right"></i>Time Critical</p>
                                    <p><i class="fas fa-caret-right"></i>Less Urgent</p>
                                </section>
                            </div>
                            <div class="item">
                                <input type="checkbox" id="item3" name="item-group">
                                <label for="item3" class="item-title">Find Dhl point <i class="fas fa-chevron-down"></i></label>
                                <section class="item-content"> 
                                    <h1>International Services to Another Country</h1>
                                    <p><i class="fas fa-caret-right"></i>Emergency</p>
                                    <p><i class="fas fa-caret-right"></i>Time Critical</p>
                                    <p><i class="fas fa-caret-right"></i>Less Urgent</p>
                                </section>
                            </div>
                            <div class="item">
                                <input type="checkbox" id="item4" name="item-group">
                                <label for="item4" class="item-title">Find a service <i class="fas fa-chevron-down"></i></label>
                                <section class="item-content"> 
                                    <h1>International Services to Another Country</h1>
                                    <p><i class="fas fa-caret-right"></i>Emergency</p>
                                    <p><i class="fas fa-caret-right"></i>Time Critical</p>
                                    <p><i class="fas fa-caret-right"></i>Less Urgent</p>
                                </section>
                            </div>
                        </section>
                    </div>
                    <div class="tab">
                        <input type="radio" id="tab2" name="tab-group">
                        <label for="tab2" class="tab-title">Logistics</label>
                        <section class="tab-content">
                            <img src="img/delivery_type_2.jpg">
                        </section>
                    </div>
                </div>
                <div id="img-with-border">
                    <img src="img/dhl_team.jpg" alt="Трое мужчин - курьеров DHL">
                </div>
            </div>
        </section>
        <section id="excellence">
            <div class="container">
                <h1>Excellence. Simply delivered.</h1>
                <p>International express deliveries; global freight forwarding by air, sea, road and rail;
                   warehousing solutions from packaging, to repairs, to storage; mail deliveries worldwide;
                   and other customized logistic services – with everything DHL does, we
                   help connect people and improve their lives.</p>
                <a href="#" class="outline-btn">learn more</a>
            </div>
        </section>
        <section id="partners">
            <div class="container">
                <a href="#"><img src="img/themeforest-light-background.png" alt="Themeforest logo"></a>
                <a href="#"><img src="img/photodune-light-background.png" alt="Photodune logo"></a>
                <a href="#"><img src="img/graphicriver-light-background.png" alt="Graphicriver logo"></a>
                <a href="#"><img src="img/codecanyon-light-background.png" alt="Codecanyon"></a>
            </div>
        </section>
        <footer>
            <div class="left-img"><img src="img/footer_bg_1.png" alt=""></div>
                <nav>
                    <section>
                        <h1>Contact Center</h1>
                        <ul>
                            <li>DHL Express</li>
                            <li>DHL Parcel & eCommerce</li>
                            <li>DHL Global Forwarding</li>
                            <li>DHL Freight</li>
                            <li>DHL Supply Chain</li>
                        </ul>
                    </section>
                    <section>
                        <h1>About Us</h1>
                        <ul>
                            <li>Partnerships</li>
                            <li>Company Portrait</li>
                            <li>DHL Global Forwarding</li>
                            <li>Green Solutions</li>
                            <li>Innovation</li>
                        </ul>
                    </section>
                    <section>
                        <h1>Social Media</h1>
                        <ul>
                            <li>Facebook</li>
                            <li>Dribbble</li>
                            <li>Behance</li>
                            <li>Gmail</li>
                            <li>Twitter</li>
                        </ul>
                    </section>
                </nav>
                <div class="right-img"><img src="img/footer_bg_2.png" alt=""></div>
        </footer>
    </body>
</html>