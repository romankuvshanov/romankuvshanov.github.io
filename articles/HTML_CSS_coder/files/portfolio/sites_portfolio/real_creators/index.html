<!DOCTYPE html>
<html lang="ru">

<head>
	<meta charset="utf-8">
	<title>Title</title>
	<meta name="description" content="">
	<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
	<link rel="shortcut icon" href="img/favicon.ico" type="image/x-icon">
    <link rel="stylesheet" href="css/main.css">
</head>

<body>
	<header class="grid" style="background-image: url(img/0-header.jpg);">
        <div class="header-content grid">
            <h1>We Are Real Creators</h1>
            <div class="header-buttons">
                <a href="#" class="button">Follow</a>
                <a href="#" class="button button-gray">Go Home</a>
            </div>
        </div>
    </header>
    <section class="portfolio-section grid">
        <div class="portfolio-item grid" style="background-image: url(img/1.jpg)">
            <span>Cool Glasses</span>
        </div>
        <div class="portfolio-item grid" style="background-image: url(img/2.jpg)">
            <span>Cool MacBook</span>
        </div>
        <div class="portfolio-item grid" style="background-image: url(img/3.jpg)">
            <span>Cool iMac</span>
        </div>
        <div class="portfolio-item grid" style="background-image: url(img/4.jpg)">
            <span>Cool Timer</span>
        </div>
        <div class="portfolio-item grid" style="background-image: url(img/5.jpg)">
            <span>Cool Hacking</span>
        </div>
        <div class="portfolio-item grid" style="background-image: url(img/6.jpg)">
            <span>Cool Bull Shit</span>
        </div>
        <div class="portfolio-item grid" style="background-image: url(img/7.jpg)">
            <span>Cool Notebook</span>
        </div>
        <div class="portfolio-item grid" style="background-image: url(img/8.jpg)">
            <span>Cool Code</span>
        </div>
        <div class="portfolio-item grid" style="background-image: url(img/9.jpg)">
            <span>Cool WorkSpace</span>
        </div>
        <div class="buttoncase">
            <a href="#" class="button">View Portfolio</a>
        </div>
    </section>
    <section class="landing-section section-ready">
        <div class="container grid">
            <div class="ready-left">
                <h2>Are You Ready?<br>Are You MTHFCKU READY??</h2>
                <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Nemo rem perferendis provident dolor nesciunt minima consectetur ipsum, architecto non inventore! Iure voluptatem, recusandae asperiores qui distinctio sunt, est error aliquam!</p>
                <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Nemo rem perferendis provident dolor nesciunt minima consectetur ipsum, architecto non inventore! Iure voluptatem, recusandae asperiores qui distinctio sunt, est error aliquam!</p>
            </div>
            
            <div class="ready-right">
                <img class="img-responsive" src="img/example.jpg" alt="Workspace with notebook">
            </div>
        </div>
    </section>
    <section class="form-section">
        <div class="container">
            <form class="grid">
                <div><input type="text" name="name1" placeholder="Your name..." required></div>
                <div><input type="text" name="name2" placeholder="Your name..."></div>
                <div><input type="text" name="name3" placeholder="Your name..."></div>
                <div><button class="button">YES! READY!</button></div>
            </form>
        </div>
    </section>
    <section class="landing-section section-blog">
        <div class="container">
            <h2>Ready To Read Our Hell Blog?</h2>
            <div class="blog-home grid">
                <a href="#" class="blog-item grid">
                    <span class="blog-item-img" style="background-image: url(img/1.jpg);"></span>
                    <span class="blog-item-content grid">
                        <h4>We Are Champions At Last Month</h4>
                        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Nemo rem perferendis provident dolor nesciunt minima consectetur ipsum, architecto...</p>
                    </span>
                </a>
                
                <a href="#" class="blog-item grid">
                    <span class="blog-item-img" style="background-image: url(img/2.jpg);"></span>
                    <span class="blog-item-content grid">
                        <h4>We Are Champions At Last Month</h4>
                        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Nemo rem perferendis provident dolor nesciunt minima consectetur ipsum, architecto...</p>
                    </span>
                </a>
                
                <a href="#" class="blog-item grid">
                    <span class="blog-item-img" style="background-image: url(img/3.jpg);"></span>
                    <span class="blog-item-content grid">
                        <h4>We Are Champions At Last Month</h4>
                        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Nemo rem perferendis provident dolor nesciunt minima consectetur ipsum, architecto...</p>
                    </span>
                </a>
                
                <a href="#" class="blog-item grid">
                    <span class="blog-item-img" style="background-image: url(img/4.jpg);"></span>
                    <span class="blog-item-content grid">
                        <h4>We Are Champions At Last Month</h4>
                        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Nemo rem perferendis provident dolor nesciunt minima consectetur ipsum, architecto...</p>
                    </span>
                </a>
                
                <a href="#" class="blog-item grid">
                    <span class="blog-item-img" style="background-image: url(img/5.jpg);"></span>
                    <span class="blog-item-content grid">
                        <h4>We Are Champions At Last Month</h4>
                        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Nemo rem perferendis provident dolor nesciunt minima consectetur ipsum, architecto...</p>
                    </span>
                </a>
                
                <a href="#" class="blog-item grid">
                    <span class="blog-item-img" style="background-image: url(img/6.jpg);"></span>
                    <span class="blog-item-content grid">
                        <h4>We Are Champions At Last Month</h4>
                        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Nemo rem perferendis provident dolor nesciunt minima consectetur ipsum, architecto...</p>
                    </span>
                </a>
                
                <div class="buttoncase">
                    <a href="#" class="button">View Blog</a>
                </div>
            </div>
        </div>
    </section>
    <footer>
        <div class="container">
            <p class="logo">We Are Real<span>Creators</span></p>
            <p class="copyright">~Â©~</p>
        </div>
    </footer>
</body>
</html>