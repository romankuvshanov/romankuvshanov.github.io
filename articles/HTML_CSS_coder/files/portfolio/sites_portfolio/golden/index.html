<!DOCTYPE html>
<html lang="en">
    <head>
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <meta charset="utf-8">
        <link rel="stylesheet" href="css/main.css">
        <title>Golden</title>
    </head>
    
    <body>
        <header>
            <div class="container">
                <div class="heading clearfix">
                    <img src="img/logo.png" alt="Логотип Golden" id="logo">
                    <nav>
                        <ul class="menu">
                            <li><a href="#">Home</a></li>
                            <li><a href="#">Services</a></li>
                            <li><a href="#">Portfolio</a></li>
                            <li><a href="#">About</a></li>
                            <li><a href="#">Contact</a></li>
                        </ul>
                    </nav>
                </div>
                
                <div class="title-and-button">
                    <h1>Welcome To Our Studio!</h1>
                    <h2>It’s nice to Meet you</h2>
                    <a class="button" href="#">Tell Me More</a>
                </div>
            </div>
        </header>
        
        <section id="services">
            <div class="container">
                <div class="titles">
                    <h1>Services</h1>
                    <h2>Proin iaculis purus consequat sem cure.</h2>
                </div>
                
                <div class="services flex-box">
                    <div class="service-item">
                        <img src="img/icon1.png" alt="Иконка корзины для покупок">
                        <h1>E-Commerce</h1>
                        <p>Proin iaculis purus consequat sem cure 
                           digni ssim. Donec porttitora entum suscipit 
                           aenean rhoncus posuere odio in tincidunt.</p>
                    </div>
                    
                    <div class="service-item">
                        <img src="img/icon2.png" alt="Иконка ноутбука с галочкой на экране">
                        <h1>Responsive Web</h1>
                        <p>Proin iaculis purus consequat sem cure 
                           digni ssim. Donec porttitora entum suscipit 
                           aenean rhoncus posuere odio in tincidunt.</p>
                    </div>
                    
                    <div class="service-item">
                        <img src="img/icon3.png" alt="Иконка закрытого замочка">
                        <h1>Web Security</h1>
                        <p>Proin iaculis purus consequat sem cure 
                           digni ssim. Donec porttitora entum suscipit 
                           aenean rhoncus posuere odio in tincidunt.</p>
                    </div>
                </div>
            </div>
        </section>
        
        <section id="portfolio">
            <div class="container">
                <div class="titles">
                    <h1>Our portfolio</h1>
                    <h2>Proin iaculis purus consequat sem cure.</h2>
                </div>
                
                <div class="works">
                    <img src="img/img1.jpg" alt="Work 1" class="col-1-3"><!--
                 --><img src="img/img2.jpg" alt="Work 2" class="col-1-3"><!--
                 --><img src="img/img3.jpg" alt="Work 3" class="col-1-3">
                </div>
            </div>
        </section>
        
        <section id="about-us">
            <div class="container">
                <div class="titles">
                    <h1>About us</h1>
                    <h2>Proin iaculis purus consequat sem cure.</h2>
                </div>
                
                    <div class="block flex-box">
                        <div class="sign col-1-3">
                            <h1>July 2010</h1>
                            <h2>Our Humble Beginnings</h2>
                            <p>
                            Proin iaculis purus consequat sem cure 
                            digni ssim. Donec porttitora entum suscipit 
                            aenean rhoncus posuere odio in tincidunt. Proin 
                            iaculis purus consequat sem cure digni 
                            ssim. Donec porttitora entum suscipit.
                            </p>
                        </div>
                        <div class="image col-1-3"><img src="img/about1.jpg" alt="Workspace"></div>
                    </div>

                    <div class="block reverse flex-box">
                        <div class="sign col-1-3">
                            <h1>January 2011</h1>
                            <h2>Facing Startup Battles</h2>
                            <p>
                            Proin iaculis purus consequat sem cure 
                            digni ssim. Donec porttitora entum suscipit 
                            aenean rhoncus posuere odio in tincidunt. Proin 
                            iaculis purus consequat sem cure digni 
                            ssim. Donec porttitora entum suscipit aenean 
                            rhoncus posuere odio in tincidunt.
                            </p>
                        </div>
                        <div class="image col-1-3"><img src="img/about2.jpg" alt="Coast"></div>
                    </div>

                    <div class="block flex-box">
                        <div class="sign col-1-3">
                            <h1>December 2012</h1>
                            <h2>Enter The Dark Days</h2>
                            <p>
                            Proin iaculis purus consequat sem cure 
                            digni ssim. Donec porttitora entum suscipit 
                            aenean rhoncus posuere odio in tincidunt. Proin 
                            iaculis purus consequat sem cure digni.
                            </p>
                        </div>
                        <div class="image col-1-3"><img src="img/about3.jpg" alt="Concert"></div>
                    </div>

                    <div class="block reverse flex-box">
                        <div class="sign col-1-3">
                            <h1>February 2014</h1>
                            <h2>Our Triumph</h2>
                            <p>
                            Proin iaculis purus consequat sem cure 
                            digni ssim. Donec porttitora entum suscipit 
                            aenean rhoncus posuere odio in tincidunt. Proin 
                            iaculis purus consequat sem cure digni 
                            ssim. Donec porttitora entum suscipit aenean.
                            </p>
                        </div>
                        <div class="image col-1-3"><img src="img/about4.jpg" alt="Hiking"></div>
                    </div>
            
            </div>
        </section>
        
        <footer>
            <div class="container">
                <p>© Copyright 2014 FreebiesXpress.com</p>
            </div>
        </footer>
    </body>
</html>